test/Microsoft.TemplateEngine.Cli.UnitTests/CacheUpgradeTests.cs

```diff
@@ -49,8 +49,8 @@ private static JObject CacheDataVersion1000
       ""Tags"": {
                 ""type"": {
                     ""Description"": null,
-          ""ChoicesAndDescriptions"": {
```

Comment:
I think this change actually breaks the test, it seems the idea of the test is that the code can still read older version of the cache.

Reply:
I see. What should be the proper action to take here, then? Revert the changes to this JSON, but also update `TemplateInfoReaderVersion1_0_0_0` to still be able to read this old format?

Reply:
It depends whether we are keeping support of reading old versions of cache or not. 

1) If we do keep support: we need to create another version (1.0.0.4) to read new format and keep old versions intact.
2) If we don't want to keep support: remove all the versions and keep implementation which supports reading just latest version of cache.